lib/react_client/component_factory.dart

```diff
@@ -36,20 +37,6 @@ dynamic listifyChildren(dynamic children) {
   }
 }
 
-/// Returns the original Dart handler function that, within [_convertEventHandlers],
-/// was converted/wrapped into the function [jsConvertedEventHandler] to be passed to the JS.
-///
-/// Returns `null` if [jsConvertedEventHandler] is `null`.
-///
-/// Returns `null` if [jsConvertedEventHandler] does not represent such a function
-///
-/// Useful for chaining event handlers on DOM or JS composite [ReactElement]s.
-Function unconvertJsEventHandler(Function jsConvertedEventHandler) {
```

Comment:
I moved this into factory_util.dart so that it would live next to event conversion logic.

I did this when I had similar unconversion logic for refs, but ended up not needing that new logic. Instead of moving `unconvertJsEventHandler` back, I thought I'd keep it in its new home.